gammapy/estimators/parameter.py

```diff
@@ -345,5 +377,104 @@ def run(self, datasets, parameter):
             if "scan" in self.selection_optional:
                 result.update(self.estimate_scan(datasets, parameter))
 
+            if "sensitivity" in self.selection_optional:
+                result.update(self.estimate_sensitivity(datasets, parameter))
+
         result.update(self.estimate_counts(datasets))
         return result
+
+
+class ParameterSensitivityEstimator:
+    """Estimate the sensitivity to a given parameter
+
+    Computes the TS distribution in the non-null hypothesis using the
+    log likelihood of the Asimov dataset (i.e. a dataset with counts = npred)
+    and the non central chi2 distribution.
+    Once the TS distribution under the testing hypothesis is known,
+    one can compute the required parameter value
+    to have 50% of measurements above a given significance threshold.
+
+
+    Parameters
+    ----------
+    parameter : `~gammapy.modeling.Parameter`
+       Parameter to test
+    null_value : float or `~gammapy.modeling.Parameter`
+        Value of the parameter for the null hypothesis.
+    n_sigma : int, default=5
```

Comment:
Having a float would be more generic...

Reply:
I would propose to change that consistently for all estimators in a separate PR.